src/style.css

```diff
@@ -1 +1,72 @@
-/* Styles go here */
+/* stylelint-disable font-family-no-missing-generic-family-keyword */
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
use * selector only for box-sizing, reset directly that element styles that you need